portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -12,12 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** Spaces out comment text from author name. */
+const commentHyphen = " -";
+
 /**
  * Fetches data from servlet and sets it in the comments section of portfolio.
  * Called whenever comments section is loaded.
  */
 async function getData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('data-container').innerText = quote;
+  const jsonData = await response.json();
+  
+  const commentsListElement = document.getElementById('comments-container');
+  commentsListElement.innerHTML = '';
+  for (comment of jsonData) {
+    commentsListElement.appendChild(createListElement(comment.commentText + commentHyphen + comment.name));
+  }
+}
+
+/** 
```

Comment:
This comment can be made more simple.

Creates and returns a <li> element containing {@code text} from comment.

Then you do not need @param or @return because it is covered in the summary.

Reply:
done